web/wp-content/plugins/ad-stack/inc/ads-global-targeting.php

```diff
@@ -32,17 +32,31 @@ function ads_global_targeting_parameters() {
 	$url_part = $unslash_url;
 
 	$g_targeting = array(
-		'property' => '6178',
-		'siteId'   => 'cpt',
-		'pageType' => $page_type,
-		'urlPath'  => $url_part,
-		'keyWords' => get_post_tags(),
-		'category' => $top_level_categories,
-		'topic'    => $sub_categories,
+		'property'   => '6178',
+		'siteId'     => 'cpt',
+		'pageType'   => $page_type,
+		'urlPath'    => $url_part,
+		'keyWords'   => get_post_tags(),
```

Comment:
This should be filterable, to allow configuration from external places (e.g. an admin screen in this same plugin, or a separate plugin that integrates with the ad stack).